UI/obs-app.cpp

```diff
@@ -1932,7 +1933,8 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 	ScopeProfiler prof{run_program_init};
 
 #if (QT_VERSION >= QT_VERSION_CHECK(5, 11, 0))
-	QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+	if (!opt_disable_high_dpi_scaling)
+		QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```

Comment:
This will probably not work as expected on newer Qt versions (6.0+) where Qt::AA_EnableHighDpiScaling is enabled by default.  You would have to explicitly disable it with Qt::AA_DisableHighDpiScaling to ensure forward compatibility.